delivery/flux/README.md

```diff
@@ -1,169 +1,205 @@
-  --private=false
+
+Finally, create a git source that uses this secret:
+
+```bash
+GITHUB_USER=<your_github_username>
+flux create source git podtato-flux-repo \
+    --url=ssh://git@github.com/${GITHUB_USER}/podtato-head \
+    --secret-ref podtato-flux-secret \
+    --branch=main
+
+# verify
+flux get source git
 ```
 
-The bootstrap command creates a repository if one doesn't exist, commits manifests for Flux Components to the default branch, and installs the flux components. Then it configures the target cluster to synchronize with the repository.
+### Process and apply
 
-### Clone the git repository
+Now that a git source is available we will instruct Flux how to render and apply
+it. Flux provides two rendering strategies - kustomizations and HelmReleases.
+Both build on the git repo source created above.
 
-We are going to drive app deployments in a GitOps manner, using the Git repository as the desired state for our cluster. Instead of applying the manifests directly to the cluster, Flux will apply it for us instead.
+#### HelmRelease
 
-Therefore, we need to clone the repository to our local machine:
+A HelmRelease composes a chart from a git or helm repository with values stored
+in the resource and applies it to the cluster.
 
-```
-$ git clone https://github.com/$GITHUB_USER/podtato-test
-$ cd podtato-test
+```bash
+# the command only reads values from files so write this to one first
+tmp_values_file=$(mktemp)
+echo -e "main:\n  serviceType: NodePort" > ${tmp_values_file}
+
+flux create helmrelease podtato-flux-release \
+    --target-namespace=podtato-flux \
+    --create-target-namespace \
+    --source=GitRepository/podtato-flux-repo.flux-system \
+    --chart=./delivery/chart \
+    --values="${tmp_values_file}"
+
+# verify
+flux get helmrelease podtato-flux-release
 ```
 
-### Adding Podtato repository to Flux
+#### Kustomization
 
-Create a GitRepository manifest `helloservice` pointing to [podtato-head][4] repository's main branch:
+A Kustomization is a render-and-apply strategy based on kustomize. Create a
+kustomization as follows:
 
-```
- flux create source git helloservice \
---url=https://github.com/cncf/podtato-head \
---branch=main \
---interval=30s \
---export > ./helloservice-source.yaml
+```bash
+kubectl create namespace podtato-kflux
+flux create kustomization podtato-flux-kustomization \
+    --target-namespace podtato-kflux \
+    --source=GitRepository/podtato-flux-repo.flux-system \
+    --path=./delivery/kustomize/base \
+    --prune=true
+
+# verify
+flux get kustomization podtato-kflux-kustomization
 ```
 
-The above command generates the following manifest:
+## Test
 
-```yaml
----
-apiVersion: source.toolkit.fluxcd.io/v1beta1
-kind: GitRepository
-metadata:
-  name: helloservice
-  namespace: flux-system
-spec:
-  interval: 30s
-  ref:
-    branch: main
-  url: https://github.com/cncf/podtato-head
-```
+### Verify delivery
 
-Commit and push the manifest to the `podtato-test` repository:
```

Comment:
It would be helpful to retain this example of _checking in_ the config change rather than running an imperative command. I'll add this back in.

Reply:
Did so in latest commit.